features/smoulder-tests/supplier/opportunities.feature

```diff
@@ -50,12 +50,11 @@ Scenario: Specialist roles are selectable for Digital specialists
   When I note the result_count
   Then a filter checkbox's associated aria-live region contains that result_count
 
-@skip-production
 Scenario Outline: Specialist roles are not selectable for non-Digital specialists lots
   Given I visit the /digital-outcomes-and-specialists/opportunities page
   Then I don't see a 'Designer' checkbox
   And I don't see any 'specialistRole' checkboxes
-  When I click '<lot>'
+  When I limit the search results to the '<lot>' category
```

Comment:
Let's not call it `category`, as that means something else in G-Cloud searches and could get confusing. The `'<lot>' lot`?

Reply:
Ah right good point

Reply:
Ideally this step would be general to both G-Cloud and DOS search pages though; I called it category initially because it's what it's called in the user interface (even though it is lots in DOS). If we called it lot would that get confusing for G-Cloud when it's not a lot?

Reply:
Can we leave it with 'click' in the name, have to dig in to the code to see what it actually does otherwise
`When I click the '<lot>' lot/category`

Reply:
I've changed it to `The I click the '<lot>' category link`